data/grib2_table_4_2_0_2.csv

```diff
@@ -12,7 +12,7 @@
 6,"MNTSF","Montgomery stream function","(m^2)/(s^2)","UC_NONE"
 7,"SGCVV","Sigma coordinate vertical velocity","1/s","UC_NONE"
 8,"VVEL","Vertical velocity (pressure)","Pa/s","UC_NONE"
-9,"DZDT","Verical velocity (geometric)","m/s","UC_NONE"
+9,"DZDT","Vertical velocity (geometric)","m/s","UC_NONE"

```

Comment:
the same change should be done in frmts/grib/degrib/data/ which is one of the sources of that generated file